src/Hangashore/lib/microcontrollers/Message.ts

```diff
@@ -0,0 +1,16 @@
+import {crc16} from './Checksum';
+
+export class Message {
+    static from(buffer: Buffer) {
+        return new Message(buffer[1], buffer.slice(2, buffer.length - 2));
+    }
+
+    constructor(readonly command: number, readonly payload: Buffer) {
+        // ???
```

Comment:
This constructor uses [Parameter properties](https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties), so `command` and `payload` become parameters